From 94272bc85c286169f20c33c353c3f5f5ae8f21de Mon Sep 17 00:00:00 2001
From: Conrad Kleinespel <conradk@conradk.com>
Date: Mon, 18 May 2015 22:03:36 +1000
Subject: [PATCH] fix compile errors on Docker Santiago image - see T80

Summary: Errors were caused by differences in support for the "const" keyword depending on the compiler version.

Test Plan: Run cassandane tests. Compile the code on different platforms.

Reviewers: #imap_reviewers

Maniphest Tasks: T80

Differential Revision: https://git.cyrus.foundation/D42
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 616d0e29d7..791528157e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -134,7 +134,6 @@ AC_PROG_INSTALL
 AC_USE_SYSTEM_EXTENSIONS
 AC_ISC_POSIX
 AC_PROG_AWK
-AC_C_CONST
 AC_SYS_LONG_FILE_NAMES
 if test $ac_cv_sys_long_file_names = no; then
 	AC_MSG_ERROR(The Cyrus IMAPD requires support for long file names)
